ppr-ui/src/store/state/state-model.ts

```diff
@@ -379,7 +379,8 @@ export const stateModel: StateModelIF = {
       },
       phoneNumber: '',
       phoneExtension: ''
-    }
+    },
+    qsSubmittingParty: null
```

Comment:
We don't need to initialize the entirety of the submitting party as this data is read only and wont be mutated.
As such, it won't need the reactive hooks created as other information that is collected.